controllers/admin_client_mock.go

```diff
@@ -532,6 +532,10 @@ func (client *mockAdminClient) GetConnectionString() (string, error) {
 	return client.Cluster.Status.ConnectionString, nil
 }
 
+func (client *mockAdminClient) GetConnectionStringFromCLI() (string, error) {
```

Comment:
Do you think we still need this one, once we get https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1421 merged? I think a better approach to using the fdbcli to determine the running version is to use the information from the status that we fetch in the next step in updateStatus.

For reference, this is the place where we check the running version: https://github.com/FoundationDB/fdb-kubernetes-operator/blob/main/controllers/update_status.go#L99-L115 with a simple heuristic (just checking what version the actual fdbserver processes are running).